index.html

```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
     </section>
 
     <!-- headline ends -->
+
+    <!-- Work section starts -->
+
+    <section class="works">
+      <section class="cards">
+        <header class="project-snippet">
+          <a class="snippet" href="#"><img src="./Properties/snippet1.png" alt=""></a>
+        </header>
+        <footer class="project-details">
+          <section class="project-details-heading">
+            <h3>TONIC</h3>
+            <ul>
+              <li class="first-list-item">CANOPY</li>
+              <li class="list-items">Back End Dev</li>
+              <li class="list-items">2015</li>
+            </ul>
+          </section>
+          <p>A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+         </p>
+         <ul class="tools">
+          <li class="tools-items">HTML</li>
+          <li class="tools-items">CSS</li>
+          <li class="tools-items">JAVASCRIPT</li>
+         </ul>
+         <button class="user-buttons">
+          See Project
+         </button>
```

Comment:
- [ ] According to the project requirement you should stick to the design as much as possible like color, font and spacing. I would suggest you please use the same background for buttons ie: `background: #FFFFFF;`
- [ ] Please implement to all buttons. Also please increase the font weight for heading Tonic and please decrese the size to it looks like figma design. PLease implement for all section.
### Your design
![his](https://user-images.githubusercontent.com/81727988/235161498-20b2c563-957a-43df-803f-eb03dce4cacd.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/235161538-5ac00eac-8b77-4af4-9d85-56cc929ca21f.png)